vignettes/admiral.Rmd

```diff
@@ -32,17 +32,22 @@ library(admiral)
 data("dm")
 data("ds")
 data("ex")
+```
 
+```{r, eval=TRUE}
 # derive treatment variables
 adsl <- dm %>%
   mutate(TRT01P = ARMCD, TRT01A = ACTARMCD) %>%
   derive_var_trtsdtm(dataset_ex = ex) %>%
-  mutate(TRTSDT = date(TRTSDTM)) %>%
   derive_var_trtedtm(dataset_ex = ex) %>%
-  mutate(TRTEDT = date(TRTEDTM)) %>%
+  derive_vars_dtm_to_dt(vars(TRTSDTM, TRTEDTM)) %>%
   derive_var_trtdurd()
 ```
 
+```{r, eval=TRUE, echo=FALSE}
+dataset_vignette(adsl)
```

Comment:
Please specify the `display_vars` parameter such that `USUBJID` and the new variables created in the example code are displayed.

Reply:
Done.